static/templates/settings/notification-settings.handlebars

```diff
@@ -20,11 +20,21 @@
               "label" settings_label.enable_stream_sounds
               "end_content" '<div class="propagate_stream_notifications_change"></div>'}}
 
-            {{partial "settings_checkbox"
-              "setting_name" "enable_stream_push_notifications"
-              "is_checked" page_params.enable_stream_push_notifications
-              "label" settings_label.enable_stream_push_notifications
-              "end_content" '<div class="propagate_stream_notifications_change"></div>'}}
+            <div class="input-group {{#unless page_params.realm_push_notifications_enabled}}control-label-disabled{{/unless}}" id="push-warning">
+                <label class="checkbox">
+                    <input type="checkbox" name="enable_stream_push_notifications" id="enable_stream_push_notifications"
+                      {{#unless page_params.realm_push_notifications_enabled}}disabled="disabled"{{/unless}}
+                      {{#if page_params.realm_push_notifications_enabled}}
+                      checked="checked"
+                      {{/if}} />
+                    <span></span>
+                </label>
+                <label for="enable_stream_push_notifications" id="enable_stream_push_notifications" class="inline-block">
+                    {{settings_label.enable_stream_push_notifications}}
+                </label>
+                <i class="fa fa-question-circle change_email_tooltip settings-info-icon" {{#if page_params.realm_push_notifications_enabled}}style="display:none"{{/if}}  data-toggle="tooltip"
+                  title="{{t 'The server is not configured for push notifications' }}"></i>
+            </div>
```

Comment:
can't this just be done by adjusting `end_content`, rather than removing the macro?

Reply:
@timabbott should I add the tweak to `settings_checkbox` for the case when there is a need for `unless` to prevent such code duplication.

Reply:
@timabbott @shubhamdhama I'm sorry but I'm not able to get what I need to change(in reference to the above comments). It would be great if you could elaborate more on it.